rules/naming/src/Rector/ClassMethod/RenameParamToMatchTypeRector.php

```diff
@@ -99,8 +105,6 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
-        $this->hasChanged = false;
-
```

Comment:
Why is this removed, when its still used bellow?

Reply:
Oooops, it was accidently removed, my bad..